### PR TITLE
Mount pull secret

### PR DESCRIPTION
**Pull request by lmilbaum**

### CODE REVIEW

File: ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml [hunk 1]
```diff
@@ -15,6 +15,9 @@ tag_specification:
   namespace: ocp
 tests:
   - as: "e2e-test"
+    secret:
+      name: openshift-cnv-dev-cnv-bot-pull-secret
+      mount_path: /etc/cnv-installer/pull-secret
```
Comment (hongkailiu) [inline ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml]: According to https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md#testsecrets

Shouldnt it be

```
    secrets:
      - name: openshift-cnv-dev-cnv-bot-pull-secret
         path: /etc/cnv-installer/pull-secret
```

**Comment by hongkailiu** [comment]
You might need to run prowgen to update job as well.

File: ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml (hunk 1 above)
Comment (lioramilbaum) [reply ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml]: you are correct. fixing it.

**Comment by lioramilbaum** [comment]
I have run the prowgen utility and it didn't update the job file

File: ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml [hunk 2]
```diff
@@ -13,6 +13,10 @@ resources:
 tag_specification:
   name: "4.3"
   namespace: ocp
+test:
```
Comment (petr-muller) [inline ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml]: There's no top-level `test` field, this belongs under the individual test item. (https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md#testsecrets)

Comment (petr-muller) [reply ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml]: LOL, there's a mistake in that doc, I'll fix it.

Comment (lioramilbaum) [reply ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml]: I have moved the secrets as you suggested, but the prowgen utility complains:
```
ERRO[0000] Failed to load CI Operator configuration      error="failed to load ci-operator config (error unmarshaling JSON: json: cannot unmarshal object into Go struct field TestStepConfiguration.tests.secrets of type []*api.Secret)" source-file=/home/lmilbaum/src/github.com/lioramilbaum/release/ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml
FATA[0000] Failed to generate jobs                       error="failed to load ci-operator config (error unmarshaling JSON: json: cannot unmarshal object into Go struct field TestStepConfiguration.tests.secrets of type []*api.Secret)" source=/home/lmilbaum/src/github.com/lioramilbaum/release/ci-operator/config subdir=openshift-cnv/cnv-ci target=/home/lmilbaum/src/github.com/lioramilbaum/release/ci-operator/jobs
```

Comment (lioramilbaum) [reply ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml]: ignore. figured it out.

Comment (petr-muller) [reply ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml]: That's likely because `secrets` is a YAML list, not an individual item.

**Pull request merged**